### PR TITLE
Added Default Value for filters

### PR DESCRIPTION
**Pull request by elfeffe**
You just have to add this line to your filter:

`    public $defaultValue = 'just_parents';
`

where "just_parents" is one of the options

```
/**
     * Defines the title and value for each option
     *
     * @return Array associative array with the title and values
     */
    public function options(): Array
    {
        return [
            'Include Children' => 'children',
            'Just parents' => 'just_parents',
        ];
    }
```

### CODE REVIEW

File: src/Views/DataView.php [hunk 1]
```diff
@@ -89,6 +92,20 @@ public function getItems(Searchable $searchable, Filterable $filterable, Sortabl
         return $query->paginate($this->paginate);
     }
 
+    public function applyDefaultFilters()
+    {
+        if(empty($this->filters))
+        {
+            foreach($this->filters() as $filter)
```
Comment (Gustavinho) [inline src/Views/DataView.php:99]: What if we have multiple filters with a default value? Will the last one be assigned?

File: src/Views/DataView.php [hunk 2]
```diff
@@ -80,6 +80,9 @@ public function getItems(Searchable $searchable, Filterable $filterable, Sortabl
         $query = $this->repository();
 
         $query = $searchable->searchItems($query, $this->searchBy, $this->search);
+
+        $this->applyDefaultFilters();
```
Comment (Gustavinho) [inline src/Views/DataView.php:84]: does it make more sense to you moving this to the `mount` function?

**Comment by Gustavinho** [comment]
Hi @elfeffe thanks for opening this PR with this feature, which is great

**Comment by elfeffe** [comment]
I think the new commit fixes the issues.

**Comment by Gustavinho** [comment]
Excellent, thanks! I'll merge it into the `release/2.3.0` branch

**Comment by Gustavinho** [comment]
Hi @elfeffe I merged it, I just added a test case to support this feature, works great!!!

image

**Comment by elfeffe** [comment]
Thank you. I didn’t do it because I don’t really know how to do tests (yes,
my mistake).

**Pull request merged**